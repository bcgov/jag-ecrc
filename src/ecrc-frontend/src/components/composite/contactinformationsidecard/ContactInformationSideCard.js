import React from "react";
import SideCard from "../../base/sideCard/SideCard";

export default function ContactInformationSideCard() {
  const contact = {
    heading: "Contact Information",
    content:
      "Criminal Records Review Program staff are available Monday to Friday from 8:30 a.m. - 4:30 p.m.",
    type: "bluegrey"
  };

  return <SideCard sideCard={contact} />;
}
