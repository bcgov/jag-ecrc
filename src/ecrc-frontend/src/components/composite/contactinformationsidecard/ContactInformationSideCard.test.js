import React from "react";
import { create } from "react-test-renderer";

import ContactInformationSideCard from "./ContactInformationSideCard";

describe("ContactInformationSideCard Component", () => {
  test("Matches the snapshot", () => {
    const contactInformation = create(<ContactInformationSideCard />);
    expect(contactInformation.toJSON()).toMatchSnapshot();
  });
});
