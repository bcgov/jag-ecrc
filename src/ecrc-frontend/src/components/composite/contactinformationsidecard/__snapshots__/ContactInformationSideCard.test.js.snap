// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactInformationSideCard Component Matches the snapshot 1`] = `
<div
  className="dashboard-spacing"
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="row"
  >
    <div
      className="col-12"
      style={
        Object {
          "paddingTop": "30px",
          "position": "relative",
        }
      }
    >
      <p />
      <section
        className="bluegrey-container"
        id="bluegrey-section"
      >
        <div
          className="container-background bluegrey-heading"
        >
          <h2
            className="heading-style"
          >
            Contact Information
          </h2>
        </div>
        <div
          className="bluegrey-content"
        >
          <p
            className="content-style"
          >
            Criminal Records Review Program staff are available Monday to Friday from 8:30 a.m. - 4:30 p.m.
            <b>
              Office:
            </b>
            Toll free - 1 855 587-0185 (press option 2)
            <b>
              Email:
            </b>
            criminalrecords@gov.bc.ca
          </p>
        </div>
      </section>
    </div>
  </div>
</div>
`;
